src/dlls/mscoree/unixinterface.cpp

```diff
@@ -353,88 +353,3 @@ int coreclr_execute_assembly(
     return hr;
 }
 
-#ifdef PLATFORM_UNIX
```

Comment:
Does this edit relate to the bug fix? I'm not opposed to cleanup, but I don't know where the bar is now. If this is an unrelated code change you might be asked to split it out into a seperate commit that won't get integrated to RTM.